docs/csharp/tuples.md

```diff
@@ -26,24 +26,26 @@ different ways to use them, and initial guidance on working with Tuples.
 
 Let's start with the reasons for adding new Tuple support. Methods return
 a single object. Tuples enable you to package multiple values in that single
-object more easily.
+object more easily. You don't need behavior on these types, but you want to
+package multiple values in a single object.
 
-The .NET Framework already had a `Tuple` generic classes. These classes,
-however, suffered from two limitations. For one, the `Tuple` classes named
+The .NET Framework already had `Tuple` generic classes. These classes,
+however, had two major limitations. For one, the `Tuple` classes named
 their fields `Item1`, `Item2`, and so on. Those names carry no semantic
-information. Using these `Tuple` types makes it harder to know the meaning
-of each of the fields. Another concern is that the `Tuple` class is a
-reference type. Using the `Tuple` type means allocating objects. On hot
+information. Using these `Tuple` types does not enable communicating the
+meaning of each of the fields. Another concern is that the `Tuple` classes are
+reference types. Using one of the `Tuple` types means allocating objects. On hot
 paths, this can have a measurable impact on your application's performance.
```

Comment:
There's one other issue: although static factory methods made it easier, one of the ways to instantiate a tuple is by calling its class constructor, which is extremely cumbersome. Feel free to ignore my comment, though, if you don't think that's worth mentioning.